### DISCUSSION TITLE
Update (05/2021)

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#29] [Archived]
Almost a month since the last commit. Yep, I took a break, which is not unusual for me, but now I'm back to work.
I'm writing this little post to show what I am working on at the moment.

https://user-images.githubusercontent.com/16880178/117538631-5d35d200-b007-11eb-826d-4ae4af7e17ee.mp4

It's a wizard/stepper control. As far as I know, no one has ever brought such control to JavaFX.
Obviously keep in mind that it is still in the experimental phase, there are still several features and improvements to be introduced however for now the development is proceeding quite well, so I plan to finish it soon.
As for the rest, before releasing the new version as I already mentioned some time ago I will separate Flowless and controls based on it into a new library. 

These will be the most important news but most likely there will be other changes as well, so keep an eye on the project 😉

**Comment by Kilobyte1000** [comment]
Its great! The closest thing I know to this is [this](https://github.com/Dansoftowner/SegmentDialogFX). But, both are quite stylistically different, and yours look richer on the full screen, while the other belongs to a dialog.

**Reply by palexdev** [reply]
Interesting, I didn't know about that library.

Mine should work well anywhere you want, pane, dialog. It is basically a BorderPane.

**Comment by palexdev** [comment]
Preview of upcoming commit (Huge update Part 2)

https://user-images.githubusercontent.com/16880178/118485249-7bdb4d80-b718-11eb-818e-08c2c507d6f0.mp4

Yep, new ripple generator is on the way, better and smoother animation, more control on ripples position and animations and many new additions.
